.github/workflows/release.yaml

```diff
@@ -9,11 +9,12 @@ on:
 
 jobs:
   release:
-    runs-on: ubuntu-20.04
+    name: Prepare github release
+    runs-on: ubuntu-24.04
     if: ${{ github.event.workflow_run.conclusion == 'success' }}
     steps:
       - name: checkout
-        uses: actions/checkout@v4.1.7
+        uses: actions/checkout@v4
```

Comment:
Given the history of security issues with github actions, it feels a little safer to be more specific about versioning, lest we pull in a vulnerable update?

Reply:
as we're not exposing any `secrets` to either the whole workflow or this step, the step does not get access to any of repo secrets leaking. also, `action/` repo houses the official github actions.